Source/Mosa.Compiler.Common/Configuration/Settings.cs

```diff
@@ -198,6 +198,21 @@ public long GetValue(string fullname, long defaultValue)
 		return defaultValue;
 	}
 
+	public ulong GetValue(string fullname, ulong defaultValue)
+	{
+		var property = GetProperty(fullname);
+
+		if (property == null)
+			return defaultValue;
+
+		if (UInt64.TryParse(property.Value, out ulong result))
```

Comment:
``ulong`` should be used instead of ``UInt64``.

Reply:
It's equivalent.